### ISSUE TITLE
Wrong value with border-radius and division

### ISSUE THREAD

**Issue by leonexcc** [scssphp/scssphp#58]
If you take the following SCSS code:
```
$test: 100px;

.test {
    width: $test/2;
    width: ($test/2);
    width: $test/2 + 0;
    width: ($test/2) + 0;

    padding-left: $test/2;
    padding-left: ($test/2);
    padding-left: $test/2 + 0;
    padding-left: ($test/2) + 0;

    border-radius: $test/2;
    border-radius: ($test/2);
    border-radius: $test/2 + 0;
    border-radius: ($test/2) + 0;
}
```
And run `npx node-sass test.scss` or `npx sass test.scss` on this it will print:
```
.test {
  width: 50px;
  width: 50px;
  width: 50px;
  width: 50px;
  padding-left: 50px;
  padding-left: 50px;
  padding-left: 50px;
  padding-left: 50px;
  border-radius: 50px;
  border-radius: 50px;
  border-radius: 50px;
  border-radius: 50px;
  border-radius: 50px;
}
```
If you install scssphp/scssphp 1.0.5 and run `vendor/bin/pscss test.scss` you will get:
```
.test {
  width: 50px;
  width: 50px;
  width: 50px;
  width: 50px;
  padding-left: 50px;
  padding-left: 50px;
  padding-left: 50px;
  padding-left: 50px;
  border-radius: 100px/2;
  border-radius: 100px/2;
  border-radius: 50px;
  border-radius: 50px; }
```
This is probably related to this: https://github.com/scssphp/scssphp/issues/34

Here is the official information on this topic in the SCSS Dokumentation: https://sass-lang.com/documentation/operators/numeric#slash-separated-values

**Comment by Cerdic** [comment]
This is kind of duplicate of #50, solved by ee2b052b7da888b6b4981365b150e8758bfcef10
But indeed it's not in the 1.0.5 release, it will comme with next one

**Comment by leonexcc** [comment]
Oh sorry, thought i looked good enough for existing ones.
